public/app/core/components/ForgottenPassword/ForgottenPassword.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC, useState } from 'react';
+import { Form, Field, Input, Button, Legend, Container, useStyles, HorizontalGroup, LinkButton } from '@grafana/ui';
+import { getBackendSrv } from '@grafana/runtime';
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+interface EmailDTO {
+  userOrEmail: string;
+}
+
+const paragraphStyles = (theme: GrafanaTheme) => css`
+  color: ${theme.colors.formDescription};
+  font-size: ${theme.typography.size.sm};
+  font-weight: ${theme.typography.weight.regular};
+  margin-top: ${theme.spacing.sm};
+  display: block;
+`;
+
+export const ForgottenPassword: FC = () => {
+  const [emailSent, setEmailSent] = useState(false);
+  const styles = useStyles(paragraphStyles);
+
+  const sendEmail = async (formModel: EmailDTO) => {
+    const res = await getBackendSrv().post('/api/user/password/send-reset-email', formModel);
+    if (res) {
+      setEmailSent(true);
+    }
+  };
```

Comment:
Think this can be moved outside of the component

Reply:
Sure, but setEmailSent still needs to be in the component